### DISCUSSION TITLE
YARP In-process

### DISCUSSION THREAD

**Discussion by ipavlu** [dotnet/yarp#948] [Q&A] [answered]
Hi,

is it possible to integrate yarp into an existing .net core app, razor mvc or blazor server side?
In-process?

Thanks

IP

**Comment by Tratcher** [comment, marked as answer]
It integrates into the ASP.NET Core route table and the most specific route match handles the request. The apps we use for testing include MVC controllers. I haven't tried it with blazor. Try it out and let us know if you have trouble.

**Reply by ipavlu** [reply]
Thanks, 
I will try, hopefully it won't take me ages to get time to get to experimenting.
IP
